### PR TITLE
Containers withFormik

### PR DESCRIPTION
**Pull request by pink-bunny**

### CODE REVIEW

File: src/pages/Login/__tests__/container.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      handleSubmit(values, { props, setErrors, setSubmitting });
+
+      expect(props.requestSession).toHaveBeenCalledWith(values, setErrors, setSubmitting);
+    });
+  });
+
+  describe('validationSchema', () => {
+    const container = wrapper.dive().props().validationSchema().describe();
+
+    expect(container).toMatchSnapshot();
```
Comment (plzen) [inline src/pages/Login/__tests__/container.test.js]: Wrap `expect` to `it` block

File: src/pages/MyLists/CreateNewListModal/__tests__/container.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+      expect(props.createMyList).toHaveBeenCalledWith(values, setErrors, props.hideModal);
+    });
+  });
+
+  describe('validationSchema', () => {
+    const container = wrapper.dive().dive().props().validationSchema()
+      .describe();
+
+    expect(container).toMatchSnapshot();
```
Comment (plzen) [inline src/pages/MyLists/CreateNewListModal/__tests__/container.test.js]: Wrap `expect` to `it` block

File: src/pages/Dashboard/Search/__tests__/container.test.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      handleSubmit(values, { props });
+
+      expect(props.searchMovies).toHaveBeenCalledWith(values.search);
+    });
+  });
+
+  describe('validationSchema', () => {
+    const container = wrapper.dive().props().validationSchema().describe();
+
+    expect(container).toMatchSnapshot();
```
[the remark this answers is not in the document]
Comment (pink-bunny) [reply src/pages/Dashboard/Search/__tests__/container.test.js]: Done.

File: src/pages/Login/__tests__/container.test.js (hunk 1 above)
Comment (pink-bunny) [reply src/pages/Login/__tests__/container.test.js]: Done.

**Pull request merged**